models/index.js

```diff
@@ -0,0 +1,9 @@
+// To make importing them easier, you can export all models from single file
+import User, { UserSchema } from './NextAuthUser';
+
+export default {
+  User: {
+    model: User,
+    schema: UserSchema,
+  },
+};
```

Comment:
I don't support doing it this way if you're only gonna use it in one place. Either we put all out models here or in [...nextAuth].js we construct the modal ourselves instead of create a separate file for it.